### PR TITLE
Fixes truncated items when list is larger than 10

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Addresses #8

### CODE REVIEW

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.spec.ts [hunk 1]
```diff
@@ -35,13 +34,48 @@ import { ChecklistState, ChecklistStateDispatcher, ChecklistStateModel } from '.
 import {
   SkyListModule,
   SkyListComponent,
-  SkyListToolbarModule
+  SkyListToolbarModule,
+  SkyListPagingModule
 } from '@skyux/list-builder';
 
 import { Observable } from 'rxjs/Observable';
 import { BehaviorSubject } from 'rxjs/BehaviorSubject';
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/list-view-checklist/list-view-checklist.component.spec.ts]: I know you didn't do these but could we clean up the import stylings while we are touching this file?

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts [hunk 2]
```diff
@@ -353,9 +353,15 @@ export class SkyListViewChecklistComponent extends ListViewComponent implements
         this.dispatcher.filtersUpdate(filters);
       });
 
-    this.dispatcher.next(
-      new ListPagingSetPageNumberAction(Number(1))
-    );
+      if (isSelected) {
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/list-view-checklist/list-view-checklist.component.ts:357]: Would you care to put some in-code comments here describing why this is being done. I think this is a spot where we won't remember why this specific code is here.

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]
@Blackbaud-AlexKingman just a couple very small nitpick things

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.spec.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/list-view-checklist/list-view-checklist.component.spec.ts]: done.

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/list-view-checklist/list-view-checklist.component.ts:357]: good suggestion. done.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]
LGTM now

File: src/app/public/modules/list-view-checklist/fixtures/list-view-checklist-pagination.component.fixture.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    { id: '2', column1: 202, column2: 'Banana', column3: 'Ben eats bananas' },
+    { id: '3', column1: 303, column2: 'Pear', column3: 'Patty eats pears' },
+    { id: '4', column1: 404, column2: 'Grape', column3: 'George eats grapes' },
+    { id: '5', column1: 505, column2: 'Banana', column3: 'Becky eats bananas' },
+    { id: '6', column1: 606, column2: 'Lemon', column3: 'Larry eats lemons' },
+    { id: '7', column1: 707, column2: 'Strawberry', column3: 'Sally eats strawberries' },
+    { id: '8', column1: 808, column2: 'Apple', column3: 'Anne eats apples'},
+    { id: '9', column1: 909, column2: 'baz', column3: 'Anne eats baz'},
+    { id: '10', column1: 1010, column2: 'bar', column3: 'Anne eats bar'},
+    { id: '11', column1: 1111, column2: 'foo', column3: 'Anne eats foo'}
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-checklist/fixtures/list-view-checklist-pagination.component.fixture.ts:29]: Thanks for updating the demo!

**Review by Blackbaud-SteveBrush — APPROVED** [review]
LGTM

**Pull request merged**